templates/pages/index.html.njk

```diff
@@ -1,6 +1,7 @@
 {% extends "../layouts/default.html.njk" %}
 
 {% from "../_macros.njk" import svgIconNew, callForSpeakersButton %}
+{% from "../_macros.njk" import svgIconNew, callForArtistsButton %}
```

Comment:
You can add `callForArtistsButton` to line 3 where we import other two macros. This way we can remove L4. ;)